addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -1963,15 +1966,19 @@ export class OdooEditor extends EventTarget {
             // <font> node. If it remains empty, it will be cleaned up later by
             // the sanitizer.
             const zws = document.createTextNode('\u200B');
-            range.startContainer.before(zws);
+            start.before(zws);
```

Comment:
That's the same node isn't it? If so, let's not change this. Remember, we're just forward-porting here, not refactoring.

Reply:
Noted! I will remember that.